examples/api/lib/widgets/basic/overflowbox.0.dart

```diff
@@ -0,0 +1,53 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// Flutter code sample for [OverflowBox].
+
+import 'package:flutter/material.dart';
+
+void main() => runApp(const OverflowBoxApp());
+
+class OverflowBoxApp extends StatelessWidget {
+  const OverflowBoxApp({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+        appBar: AppBar(title: const Text('OverflowBox Sample')),
+        body: const Center(
+          child: OverflowBoxExample(),
+        ),
+      ),
+    );
+  }
+}
+
+class OverflowBoxExample extends StatelessWidget {
+  const OverflowBoxExample({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Center(
```

Comment:
Maybe convert to a `Column` and add a sibling widget to show that the overflow draws over siblings instead of expanding the bounding box?

```dart
  Widget build(BuildContext context) {
    return Column(
      mainAxisAlignment: MainAxisAlignment.center,
      children: <Widget>[
        const Text('Cover Me'),
        // This parent container has fixed width and
        // height of 100 pixels.
        Container(
          width: 100,
          height: 100,
          color: Theme.of(context).colorScheme.secondaryContainer,
          // This OverflowBox imposes its own constraints of maxWidth
          // and maxHeight of 200 pixels on its child which allows the
          // child to overflow the parent container.
          child: const OverflowBox(
            maxWidth: 200,
            maxHeight: 200,
            // Without the OverflowBox, the child widget would be
            // constrained to the size of the parent container
            // and would not overflow the parent container.
            child: FlutterLogo(size: 200),
          ),
        ),
      ],
    );
  }
```

Reply:
Sounds good!